extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -970,7 +970,7 @@ public function fetchDiscussionPostsNumberFrom( $user ) {
 		global $wgEnableDiscussions;
 
 		$this->setVal( 'discussionPostsCountInUserIdentityBoxEnabled', $wgEnableDiscussions );
-		if ( $wgEnableDiscussions ) {
+		if ( $wgEnableDiscussions && $user->isLoggedIn() ) {
```

Comment:
Although correct, this method is a bit odd looking in this context considering this is not the session user. For readability's sake, I prefer `!$user->isAnon()` (which calls `isLoggedIn` 😸 )